test/chrono-test.cc

```diff
@@ -1011,4 +1011,16 @@ TEST(chrono_test, glibc_extensions) {
 TEST(chrono_test, out_of_range) {
   auto d = std::chrono::duration<unsigned long, std::giga>(538976288);
   EXPECT_THROW((void)fmt::format("{:%j}", d), fmt::format_error);
+}
+
+TEST(chrono_test, year_month_day) {
+  auto loc = get_locale("es_ES.UTF-8");
+  std::locale::global(loc);  
+  auto year = fmt::year(2024 - 1900);
```

Comment:
I think std::chrono::year doesn't require the shift by 1900 and we should do the same.